__tests__/integration.spec.js

```diff
@@ -85,6 +86,52 @@ describe('toMatchImageSnapshot', () => {
     });
   });
 
+  describe('updates', () => {
+    const customSnapshotIdentifier = 'integration-update';
+    const updateImageData = fs.readFileSync(fromStubs('TestImageUpdate1pxOff.png'));
+    const updateImageSnapshotPath = path.join(__dirname, '__image_snapshots__', `${customSnapshotIdentifier}-snap.png`);
+
+    beforeEach(() => {
+      fs.writeFileSync(updateImageSnapshotPath, imageData);
+    });
+
+    afterAll(() => {
```

Comment:
Why is this needed?

Reply:
Because we want the fs to be reset to the correct original state even if the last test were to change the file via `updatePassedSnapshots: true`, so the git changes are clean.